Automattic/woocommerce-payments
# Set currency_format locale data into a transient.

Fixes #2397 

#### Changes proposed in this Pull Request

* Set the `currency_format` data into a transient that expires daily so that it will be cached for future calls.
* Add test to check that transient is set.

#### Testing instructions

* View any shop page on the front end.
* Check your `wp_options` table for `_transient_wcpay_multi_currency_locale_data`, or use xdebug to step through the code to see that the transient is being set and retrieved.

-------------------
